bedrock/base/templates/base-pebbles.html

```diff
@@ -3,7 +3,7 @@
 <!doctype html>
 {# Note the "windows" class, without javascript platform-specific
      assets default to windows #}
-<html class="windows x86 no-js" lang="{{ LANG|replace('en-US', 'en') }}" dir="{{ DIR }}" data-latest-firefox="{{ latest_firefox_version }}" data-esr-versions="{{ esr_firefox_versions|join(' ') }}" {% if settings.OPTIMIZELY_PROJECT_ID %}data-optimizely-project-id="{{ settings.OPTIMIZELY_PROJECT_ID }}"{% endif %} {% if settings.GTM_CONTAINER_ID %}data-gtm-container-id="{{ settings.GTM_CONTAINER_ID }}"{% endif %} {% block gtm_page_id %}{% endblock %} {% if settings.STUB_ATTRIBUTION_RATE %}data-stub-attribution-rate="{{ settings.STUB_ATTRIBUTION_RATE }}"{% endif %}>
+<html class="windows x86 no-js" lang="{{ LANG|replace('en-US', 'en') }}" dir="{{ DIR }}" data-latest-firefox="{{ latest_firefox_version }}" data-esr-versions="{{ esr_firefox_versions|join(' ') }}" {% if settings.STATCOUNTER_PROJECT_ID %}data-statcounter-project-id="{{ settings.STATCOUNTER_PROJECT_ID }}"{% endif %} {% if settings.STATCOUNTER_SECURITY_ID %}data-statcounter-security-id="{{ settings.STATCOUNTER_SECURITY_ID }}"{% endif %} {% if settings.OPTIMIZELY_PROJECT_ID %}data-optimizely-project-id="{{ settings.OPTIMIZELY_SECURITY_ID }}"{% endif %} {% if settings.OPTIMIZELY_PROJECT_ID %}data-optimizely-project-id="{{ settings.OPTIMIZELY_PROJECT_ID }}"{% endif %} {% if settings.GTM_CONTAINER_ID %}data-gtm-container-id="{{ settings.GTM_CONTAINER_ID }}"{% endif %} {% block gtm_page_id %}{% endblock %} {% if settings.STUB_ATTRIBUTION_RATE %}data-stub-attribution-rate="{{ settings.STUB_ATTRIBUTION_RATE }}"{% endif %}>
```

Comment:
Looks like some copy pasta here - `data-optimizely-project-id` is included twice, with the first incorrectly referencing `settings.OPTIMIZELY_SECURITY_ID`.